### DISCUSSION TITLE
When will be update?

### DISCUSSION THREAD

**Discussion by sakulmore** [vilhelmprytz/pterodactyl-installer#332] [General]
Hello, when will be the pterodactyl panel updated to version 1.8.1?

**Comment by vilhelmprytz** [comment]
This script always downloads and installs the latest available version of Pterodactyl.

**Reply by sakulmore** [reply]
And is there way to update pterodactyl? Or I must reinstall it?

**Reply by vilhelmprytz** [reply]
You can update it. Refer to the official documentation ([panel](https://pterodactyl.io/panel/1.0/updating.html) and [wings](https://pterodactyl.io/wings/1.0/upgrading.html)).

**Reply by sakulmore** [reply]
Yeah, i know that.

When will be this added? https://github.com/vilhelmprytz/pterodactyl-installer/issues/94 is it in the project to-do list.
